docs/haiku/contribution.rst

```diff
@@ -0,0 +1,23 @@
+Contributions
+#############
+
+Contributions must be made via pull request to the `main` branch.
+
+All code should have documentation, unit tests and follow the coding guidelines.
+
+Python < 3.6 is not allowed or supported.
+
+Adding Tools
+============
+
+  * Python libraries should be added to the `haiku/python_requirements.txt`
+  * Third party tools available via yum should be added to the `haiku/yum_requirements.txt`
+  * Third party tools that require custom installation should have an installation script added to the `haiku/dependencies/scripts/` directory.  The script filename must start with `install_` in order to be executed correctly
```

Comment:
```suggestion
* Python libraries should be added to the `haiku/python_requirements.txt`
* Third party tools available via yum should be added to the `haiku/yum_requirements.txt`
* Third party tools that require custom installation should have an installation script added to the `haiku/dependencies/scripts/` directory.
  The script filename must start with `install_` in order to be executed correctly
```
no indent, otherwise is different formatting